src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/AspNetCore6Plus/ResponseStreamWrapper.cs

```diff
@@ -129,17 +124,14 @@ private bool IsHtmlResponse(bool forceReCheck = false)
                 return false;
 
             // Requirements for script injection:
-            // * has to have result body
-            // * 200 or 500 response
             // * text/html response
             // * UTF-8 formatted (explicit or no charset)
 
             _isHtmlResponse =
-                _context.Response.StatusCode is 200 or 500 &&
```

Comment:
Do we not need to check for these cases?

Reply:
I don't think so; there are other valid status codes that we might want to instrument.